src/EnergyPlus/MoistureBalanceEMPDManager.cc

```diff
@@ -599,7 +600,7 @@ namespace MoistureBalanceEMPDManager {
 		}
 		// Calculate resistance between surface-layer/air interface and center of surface layer. [s/m]
 		// This is the physical surface of the material.
-		RSurfaceLayer = 1.0 / hm_surf_layer - 1.0 / h_mass_conv_in_fd - Rcoating;
+		RSurfaceLayer = 1.0 / hm_surf_layer - 1.0 / h_mass_conv_in_fd;
```

Comment:
The subtraction is meant to find only a portion of the overall resistance, such that we can calculate the humidity ratio at the actual surface (with the material resistance on one side, and the air boundary layer on the other side). This is required because the other part of the model (HeatBalanceSurfaceManager) uses this surface condition (rather than the condition inside the surface) to calculate moisture transfer from the surface to the zone air.

The mistake that was corrected was that I originally also subtracted the paint-layer resistance, but this is no included in the HeatBalanceSurfaceManager calculations, so it needed to be removed.